src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -732,7 +739,9 @@ export default {
 			 *
 			 * @type {Event}
 			 */
+			// eslint-disable-next-line vue/require-explicit-emits
 			this.$emit('figure-click', e)
+			this.$attrs.onFigureClick(e)
```

Comment:
It calls the handler twice: once as a component event handler by `$emit` and once directly in `onFigureClick`.

Reply:
I was under the impression it does not emit when it's not listed under `emits`. I removed `$attrs.onFigureClick` and it seems to work correct now.